pkg/controllers/user/logging/utils/wrap.go

```diff
@@ -226,6 +247,15 @@ func getDateFormat(dateformat string) string {
 	return "%Y.%m.%d"
 }
 
+func testReachable(network string, url string) error {
+	timeout := time.Duration(10 * time.Second)
+	_, err := net.DialTimeout(network, url, timeout)
```

Comment:
This will leak file descriptors as you aren't closing the connection on successful connect.